app/views/unattended/provisioning_templates/snippet/first_boot_setup.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: snippet
+name: first_boot_setup
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Post replacement.
+-%>
+<%= save_to_file('/etc/systemd/system/first_boot_setup_service.service',
+                 snippet('first_boot_setup_service')) %>
+<%= save_to_file('/root/first_boot_setup_script.sh',
+                 snippet('first_boot_setup_script')) %>
+systemctl enable first_boot_setup_service
+chmod +x '/root/first_boot_setup_script.sh'
```

Comment:
Do we want to have it in the `/root/` folder?
Won't be `/tmp` better for this use case?

Reply:
I have already moved the code directly to the kickstart template.